sdk/constants/futures.ts

```diff
@@ -285,3 +287,40 @@ export const BPS_CONVERSION = 10000;
 export const DEFAULT_DESIRED_TIMEDELTA = 0;
 
 export const AGGREGATE_ASSET_KEY = '0x';
+
+// subgraph fragments
+export const ISOLATED_MARGIN_FRAGMENT = gql`
+	query userFuturesMarginTransfers($walletAddress: String!) {
+		futuresMarginTransfers(
+			where: { account: $walletAddress }
+			orderBy: timestamp
+			orderDirection: desc
+			first: 10000
+		) {
+			id
+			timestamp
+			account
+			market
+			size
+			asset
+			txHash
+		}
+	}
+`;
+
+export const CROSS_MARGIN_FRAGMENT = gql`
```

Comment:
Nice, I started to move subgraph calls into queries to avoid the futures service getting too bloated but maybe this is a better alternative.

Reply:
Ah, I didn't see this. Yeah I could do either convention, I was just trying to avoid specifying the query inside the service like you mentioned. Probably better I follow the convention you started than to do this though.

Reply:
Fragments felt out of place inside constants